extension/src/test/ExternalResources.test.ts

```diff
@@ -47,23 +46,80 @@ suite("External Resources Tests", function () {
 
   test("ExternalResource.url()", function () {
     const extResource = new ExternalResource("sv-se.json", href);
-    assert.equal(href, fullUrl, "href is not correct");
-    assert.equal(extResource.url().href, href, "Unexpected url");
-    assert.equal(extResource.url().hostname, hostname, "Unexpected Hostname");
-    assert.equal(extResource.url().pathname, pathname, "Unexpected path");
-    assert.equal(extResource.url().search, search, "Unexpected search params");
+    assert.deepStrictEqual(href, fullUrl, "href is not correct");
+    assert.deepStrictEqual(extResource.url().href, href, "Unexpected url");
+    assert.deepStrictEqual(
+      extResource.url().hostname,
+      hostname,
+      "Unexpected Hostname"
+    );
+    assert.deepStrictEqual(
+      extResource.url().pathname,
+      pathname,
+      "Unexpected path"
+    );
+    assert.deepStrictEqual(
+      extResource.url().search,
+      search,
+      "Unexpected search params"
+    );
   });
 
   test("AzureBlobContainer.getBlobs()", async function () {
-    // Only run in GitHub Workflow
-    if (!process.env.GITHUB_ACTION) {
+    if (!WORKFLOW) {
       this.skip();
     }
     this.timeout(TIMEOUT);
     const exportPath = path.resolve(__dirname);
     const blobContainer = new BlobContainer(exportPath, baseUrl, sasToken);
     blobContainer.addBlob("sv-se.json");
     const result = await blobContainer.getBlobs();
-    assert.equal(result, 1, "Unexpected number of files downloaded");
+    assert.deepStrictEqual(
+      result.succeded.length,
+      1,
+      "Unexpected number of files downloaded"
+    );
+  });
+
+  test("#190 - Unexpected end of JSON input", async function () {
+    // github.com/jwikman/nab-al-tools/issues/190
+
+    if (!WORKFLOW) {
+      this.skip();
+    }
+
+    this.timeout(TIMEOUT); // Take some time to download blobs on Ubuntu... and windows!
+    const langCode = {
+      corrupt: "en-au_broken",
+      pristine: "sv-se",
```

Comment:
It'll take quite some time for me to get used to the trailing commas in the last item...

Reply:
Hehe, if it weren't for the auto format it wouldn't be there :robot: